src/Altinn.Apps/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -165,6 +165,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseDefaultSecurityHeaders();
             app.UseRouting();
+            app.UseStaticFiles('/' + applicationId);
```

Comment:
This is probably a good change as well. @lorang92 Do you see any downsides of setting this in the template?